src/e2e/specs/breachResolution.spec.ts

```diff
@@ -110,45 +42,41 @@ test.describe("Breaches Dashboard - Headers", () => {
     await settingsPage.open();
 
     // verify logo and profile button
-    await expect(async () => {
-      expect(
-        await dataBreachPage.dataBreachesLogoLink.first().getAttribute("href"),
-      ).toBe("/user/breaches");
-    }).toPass();
+    expect(
+      await dataBreachPage.dataBreachesLogoLink.first().getAttribute("href"),
+    ).toBe("/user/breaches");
 
     await page.waitForLoadState("networkidle");
     await dataBreachPage.dataBreachesNavbarProfile.click();
 
-    // verify mnanage your ff account link, settings option, help and support option, sign out option
-    await expect(async () => {
-      // menu is open
-      expect(await dataBreachPage.profileMenuExpanded()).toBeTruthy();
-
-      // menu header
-      await expect(
-        dataBreachPage.dataBreachesNavbarProfileMenuHeader,
-      ).toBeVisible();
-
-      // head text
-      expect(
-        await dataBreachPage.dataBreachesNavbarProfileMenuHeaderSubtitle.textContent(),
-      ).toEqual("Manage your Mozilla account");
-
-      // check settings
-      await expect(
-        dataBreachPage.dataBreachesNavbarProfileMenuSettings,
-      ).toBeVisible();
-
-      // help and support
-      await expect(
-        dataBreachPage.dataBreachesNavbarProfileMenuHelpAndSupport,
-      ).toBeVisible();
-
-      // sign out
-      await expect(
-        dataBreachPage.dataBreachesNavbarProfileMenuSignOut,
-      ).toBeVisible();
-    }).toPass();
+    // verify manage your Mozilla account link, settings option, help and support option, sign out option
+    // menu is open
+    expect(await dataBreachPage.profileMenuExpanded()).toBeTruthy();
```

Comment:
It's good practice to check for literal booleans, rather than type-casting it to a boolean. One day, it will avoid accidentally interpreting a non-empty string as `true` :grin: 

```suggestion
    expect(await dataBreachPage.profileMenuExpanded()).toBe(true);
```